File: C Program Files (x86) Microsoft Visual Studio 12.0 Common7 IDE WebTemplates/WebForms/CSharp/1033/Rules/WebFormsBasic.Multi/Models/TenantDbContext.cs
using System;
using System.Collections.Generic;
using System.Data.Entity;
using System.Linq;
using System.Web;

namespace $safeprojectname$.Models
{
    public class TenantDbContext : DbContext
    {
        public TenantDbContext()
            : base("DefaultConnection")
        {
        }

        public DbSet<Tenant> Tenants { get; set; }

        public DbSet<IssuingAuthorityKey> IssuingAuthorityKeys { get; set; }

        public DbSet<SignupToken> SignupTokens { get; set; }
    }
}

File: C Program Files (x86) Microsoft Visual Studio 12.0 Common7 IDE WebTemplates/MVC/VisualBasic/1033/MvcBasicApplicationv5.0/_OrgAuth/_MultiOrg/Models/TenantDbContext.vb
Imports System.Data.Entity

Public Class TenantDbContext
    Inherits DbContext
    Public Sub New()
        MyBase.New("DefaultConnection")
    End Sub

    Public Property Tenants As DbSet(Of Tenant)
    Public Property IssuingAuthorityKeys As DbSet(Of IssuingAuthorityKey)
    Public Property SignupTokens As DbSet(Of SignupToken)
End Class
